ietf/templates/nomcom/nomcom_public_base.html

```diff
@@ -48,21 +51,24 @@ <h1>
         </li>
         {% if user|is_chair_or_advisor:year or user|has_role:"Secretariat" %}
             <li class="nav-item dropdown">
-                <a class="nav-link dropdown-toggle" data-bs-toggle="dropdown" href="#">Volunteers</a>
+                <a class="nav-link dropdown-toggle {% if selected in 'eligible,volunteers' %}active{% endif %}" data-bs-toggle="dropdown" href="#">Volunteers</a>
```

Comment:
Similar issue to before, though less likely to be surprising since the second match doesn't contain the first.